eskip/eskip.go

```diff
@@ -151,6 +152,8 @@ func (t BackendType) String() string {
 		return "shunt"
 	case LoopBackend:
 		return "loopback"
+	case DynamicBackend:
+		return "dynamic"
 	default:
 		return "unknown"
```

Comment:
"unknown" should be a const and might be reused at  https://github.com/zalando/skipper/pull/918/files#diff-f983044d09504437f499749382321472R110 (maybe I am wrong and it's something else but looks similar)